openstack/barbican/templates/scripts-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: barbican-scripts
+  labels:
+    system: openstack
+    type: configuration
+    component: barbican
+
+data:
+  barbican-db-check.sh: |
+{{ include (print .Template.BasePath "/scripts/_barbican-db-check.sh.tpl") . | indent 4 }}
```

Comment:
Why is it called `db-check`? Currently, it contains the secret move. Would a more generic `barbican-nanny` maybe be more appropriate?

Reply:
i preferred `move-secrets` for container name as the pod name is barbican-nanny.